Makefile

```diff
@@ -48,6 +48,24 @@ run-acceptance-tests-base:
 run-acceptance-tests-actual:
 	$(call run_acceptance_tests,actual)
 
+define selected_acceptance_tests
```

Comment:
Why do we maintain two almost identical Makefiles?

Reply:
One is for us, the other one is for projects (the one in project-base).